src/java.base/share/classes/java/util/concurrent/ThreadPoolExecutor.java

```diff
@@ -1544,11 +1551,13 @@ else if (delta > 0) {
             // As a heuristic, prestart enough new workers (up to new
             // core size) to handle the current number of tasks in
             // queue, but stop if queue becomes empty while doing so.
+            /*
             int k = Math.min(delta, workQueue.size());
             while (k-- > 0 && addWorker(null, true)) {
                 if (workQueue.isEmpty())
                     break;
             }
+            */
```

Comment:
Is this meant to be commented out?

Reply:
This code was also commented out in the original CVS repo; here's the diff: http://gee.cs.oswego.edu/cgi-bin/viewcvs.cgi/jsr166/src/main/java/util/concurrent/ThreadPoolExecutor.java?r1=1.193&r2=1.194

The message for the `1.194` revision suggests we should NOT expect code changes at all. I've double-checked my patch, which is at least partially responsible for the `1.194` revision, and couldn't find that commenting out part.